pipelines/crc-qe-virtual-fedora.yaml

```
@@ -0,0 +1,201 @@
+        - name: e2e-cleanup-target
+          value: "true" 
+        - name: run-integration
+          value: "true"
+        - name: integration-tag
+          value: "! microshift-preset"
+        - name: integration-cleanup-target
+          value: "true" 
+      timeout: "5h" 
+    - name: s3-upload-results
+      runAfter:
+      - qe
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)